lib/contentProfileListener.js

```diff
@@ -99,8 +99,10 @@ class ContentProfileImporter extends SHRContentProfileParserListener {
     if (currentElement) {
       for (const name of names) {
         let parentElements = this.getRecursiveBasedOns(currentElement.identifier);
-        
-        if (name === 'Value' || name === '_Value') {
+        if (PRIMITIVES.includes(name)) {
```

Comment:
This is the handling I added to support adding primitives into the path. It only supports primitives as the last element in the path...but that makes sense because they would never have children.